docs/commands/json.merge.md

```diff
@@ -1 +1,115 @@
-Sets or updates the JSON value at a path.
+Merge one or more `value` in a JSON document adding, updating, deleting and/or replacing a array at `path` in `key`
+
+Compliance with Json Merge Patch [RFC7396](https://datatracker.ietf.org/doc/html/rfc7396)
+
+[Examples](#examples)
+
+## Required arguments
+
+<details open><summary><code>key</code></summary>
+
+is key to merge.
+</details>
+
+<details open><summary><code>path</code></summary>
+
+is JSONPath to specify. Default is root `$`. For new Redis keys the `path` must be the root. For existing keys, when the entire `path` exists, the value that it contains is replaced with the `json` value. For existing keys, when the `path` exists, except for the last element, a new child is added with the `json` value.
+
+</details>
+
+<details open><summary><code>value</code></summary>
+
+is value to set at the specified path. `null` value delete the path. An array as value will replace the previous array
```

Comment:
delete --> deletes.

"An array as value will replace" 
--> When `value` is an array - the previous value...